models/models.go

```diff
@@ -127,8 +127,8 @@ func getCustomLabels(field []byte) (map[string]string, error) {
 	return m, nil
 }
 
-// setCustomLabels encodes custom labels m to the Node/Service/Agent field.
-func setCustomLabels(m map[string]string, field *[]byte) error {
+// setStringMap encodes string map as model's byte array field.
+func setStringMap(m map[string]string, field *[]byte) error {
```

Comment:
⚠️ `prepareLabels` validates that map keys are valid Prometheus labels. That's not a function any string map.

Let's call it getLabels (annotations are basically labels too)

Reply:
Done